react-components/src/architecture/base/concreteCommands/pointCloud/SetPointColorTypeCommand.test.ts

```diff
@@ -0,0 +1,55 @@
+import { SetPointColorTypeCommand } from './SetPointColorTypeCommand';
+import { createFullRenderTargetMock } from '../../../../../tests/tests-utilities/fixtures/createFullRenderTargetMock';
```

Comment:
should be imported using fixtures

```suggestion
import { createFullRenderTargetMock } from '#test-utils/fixtures/createFullRenderTargetMock';
```